test/test_code_to_gast_conditionals.py

```diff
@@ -45,10 +49,12 @@ def test_else(self):
             }
         self.assertEqual(expected_gast, py_main.py_to_gast(py_code))
         self.assertEqual(expected_gast, js_main.js_to_gast(js_code))
+        self.assertEqual(expected_gast, java_main.java_to_gast(java_code))
    
     def test_elif(self):
         py_code = 'if (1):\n\tprint("1 is true")\nelif (2):\n\tprint("2 is true")\n\tprint("second line")\n'
         js_code = 'if (1) {\n\tconsole.log("1 is true")\n} else if (2) {\n\tconsole.log("2 is true")\n\tconsole.log("second line")\n}'
+        java_code = 'if (1) {System.out.println("1 is true");} else if (2) {System.out.println("2 is true"); System.out.println("second line");}'
```

Comment:
Is indenting supported for java?

Reply:
Yes for gast_to_code, but it is not necessary when testing code to gast. Java parser works the same way with and without indentation